cli/cmd/vulnerability.go

```diff
@@ -324,6 +314,55 @@ func pollScanStatus(requestID string) error {
 	}
 }
 
+// Build the cli output for vuln ctr 'scan-status', 'scan' and 'show-assessment' commands
+func buildReports(assessment *api.VulnContainerAssessment) error {
+	summaryReport := buildVulnerabilitySummaryReportTable(assessment)
+	details := vulnerabilityDetailsReport{
+		VulnerabilityDetails: filterVulContainerImageLayers(assessment.Image),
+		Packages:             filterVulnContainerImagePackages(assessment.Image),
+	}
+	detailsReport := buildVulnerabilityDetailsReportTable(details)
+	filteredAssessment := assessment
+	filteredAssessment.Image.ImageLayers = details.VulnerabilityDetails.ImageLayers
+
+	if cli.JSONOutput() {
+		if err := cli.OutputJSON(filteredAssessment); err != nil {
+			return err
+		}
+	} else {
+		cli.OutputHuman(buildReportTable(summaryReport, detailsReport))
+	}
+
+	if vulCmdState.Html {
+		if err := generateVulnAssessmentHTML(assessment); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+type vulnerabilityDetailsReport struct {
+	VulnerabilityDetails filteredImageTable
+	Packages             filteredPackageTable
+}
+
+func buildReportTable(summary string, details string) string {
```

Comment:
same here, `buildReportTable()` is too generic.

Reply:
Will fix this one too, thanks!